sdks/python/apache_beam/runners/worker/operations.py

```diff
@@ -808,8 +816,14 @@ def __init__(
     self.user_state_context = user_state_context
     self.tagged_receivers = None  # type: Optional[_TaggedReceivers]
     # A mapping of timer tags to the input "PCollections" they come in on.
+    # Force clean rebuild
```

Comment:
Let's not forget to remove this. Note: you can close and reopen PRs to rerun a test suite.